2025-09-17 00:31:31,473 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 00:36:29,187 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 00:48:25,893 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 00:57:03,275 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:02:29,055 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:10:23,916 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:25:48,838 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:31:54,786 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:42:38,368 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:56:52,621 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:56:52,622 - ingestion_db - ERROR - load_raw_data:62 - Could not establish database connection. Exiting.
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:68 - ==================================================
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:69 - Starting CSV ingestion process
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:70 - Source directory: D:\Elevated_Lab\Customer Churn Analysis for Telecom Industry\Customer-Churn-Prediction\data
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:71 - ==================================================
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:80 - Found 1 CSV file to process
2025-09-17 01:56:52,622 - ingestion_db - INFO - load_raw_data:87 - Processing file: telecom_data.csv
2025-09-17 01:56:53,192 - ingestion_db - INFO - load_raw_data:91 - Loaded CSV with shape: (7043, 21)
2025-09-17 01:56:53,192 - ingestion_db - INFO - load_raw_data:92 - Columns: ['customerID', 'gender', 'SeniorCitizen', 'Partner', 'Dependents', 'tenure', 'PhoneService', 'MultipleLines', 'InternetService', 'OnlineSecurity', 'OnlineBackup', 'DeviceProtection', 'TechSupport', 'StreamingTV', 'StreamingMovies', 'Contract', 'PaperlessBilling', 'PaymentMethod', 'MonthlyCharges', 'TotalCharges', 'Churn']
2025-09-17 01:56:53,192 - ingestion_db - ERROR - load_raw_data:114 - Unexpected error processing file telecom_data.csv: cannot access local variable 'engine' where it is not associated with a value
2025-09-17 01:56:53,192 - ingestion_db - INFO - load_raw_data:131 - ==================================================
2025-09-17 01:56:53,192 - ingestion_db - INFO - load_raw_data:132 - INGESTION COMPLETE
2025-09-17 01:56:53,192 - ingestion_db - INFO - load_raw_data:133 - Total files processed: 1
2025-09-17 01:56:53,193 - ingestion_db - INFO - load_raw_data:134 - Successful: 0
2025-09-17 01:56:53,193 - ingestion_db - INFO - load_raw_data:135 - Failed: 1
2025-09-17 01:56:53,193 - ingestion_db - INFO - load_raw_data:136 - Total time taken: 0.01 minutes
2025-09-17 01:56:53,193 - ingestion_db - INFO - load_raw_data:137 - ==================================================
2025-09-17 01:59:20,968 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 01:59:20,969 - ingestion_db - ERROR - load_raw_data:62 - Could not establish database connection. Exiting.
2025-09-17 01:59:20,969 - ingestion_db - INFO - load_raw_data:68 - ==================================================
2025-09-17 01:59:20,969 - ingestion_db - INFO - load_raw_data:69 - Starting CSV ingestion process
2025-09-17 01:59:20,969 - ingestion_db - INFO - load_raw_data:70 - Source directory: D:\Elevated_Lab\Customer Churn Analysis for Telecom Industry\Customer-Churn-Prediction\data
2025-09-17 01:59:20,969 - ingestion_db - INFO - load_raw_data:71 - ==================================================
2025-09-17 01:59:20,970 - ingestion_db - INFO - load_raw_data:80 - Found 1 CSV file to process
2025-09-17 01:59:20,970 - ingestion_db - INFO - load_raw_data:87 - Processing file: telecom_data.csv
2025-09-17 01:59:21,014 - ingestion_db - INFO - load_raw_data:91 - Loaded CSV with shape: (7043, 21)
2025-09-17 01:59:21,015 - ingestion_db - INFO - load_raw_data:92 - Columns: ['customerID', 'gender', 'SeniorCitizen', 'Partner', 'Dependents', 'tenure', 'PhoneService', 'MultipleLines', 'InternetService', 'OnlineSecurity', 'OnlineBackup', 'DeviceProtection', 'TechSupport', 'StreamingTV', 'StreamingMovies', 'Contract', 'PaperlessBilling', 'PaymentMethod', 'MonthlyCharges', 'TotalCharges', 'Churn']
2025-09-17 01:59:21,015 - ingestion_db - ERROR - load_raw_data:114 - Unexpected error processing file telecom_data.csv: cannot access local variable 'engine' where it is not associated with a value
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:131 - ==================================================
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:132 - INGESTION COMPLETE
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:133 - Total files processed: 1
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:134 - Successful: 0
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:135 - Failed: 1
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:136 - Total time taken: 0.00 minutes
2025-09-17 01:59:21,016 - ingestion_db - INFO - load_raw_data:137 - ==================================================
2025-09-17 02:06:22,089 - ingestion_db - ERROR - create_connection:22 - Failed to create database connection: (pyodbc.OperationalError) ('08001', '[08001] [Microsoft][ODBC Driver 17 for SQL Server]Named Pipes Provider: Could not open a connection to SQL Server [53].  (53) (SQLDriverConnect); [08001] [Microsoft][ODBC Driver 17 for SQL Server]Login timeout expired (0); [08001] [Microsoft][ODBC Driver 17 for SQL Server]A network-related or instance-specific error has occurred while establishing a connection to SQL Server. Server is not found or not accessible. Check if instance name is correct and if SQL Server is configured to allow remote connections. For more information see SQL Server Books Online. (53)')
(Background on this error at: https://sqlalche.me/e/20/e3q8)
2025-09-17 02:06:22,090 - ingestion_db - ERROR - load_raw_data:62 - Could not establish database connection. Exiting.
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:68 - ==================================================
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:69 - Starting CSV ingestion process
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:70 - Source directory: D:\Elevated_Lab\Customer Churn Analysis for Telecom Industry\Customer-Churn-Prediction\data
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:71 - ==================================================
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:80 - Found 1 CSV file to process
2025-09-17 02:06:22,090 - ingestion_db - INFO - load_raw_data:87 - Processing file: telecom_data.csv
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:91 - Loaded CSV with shape: (7043, 21)
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:92 - Columns: ['customerID', 'gender', 'SeniorCitizen', 'Partner', 'Dependents', 'tenure', 'PhoneService', 'MultipleLines', 'InternetService', 'OnlineSecurity', 'OnlineBackup', 'DeviceProtection', 'TechSupport', 'StreamingTV', 'StreamingMovies', 'Contract', 'PaperlessBilling', 'PaymentMethod', 'MonthlyCharges', 'TotalCharges', 'Churn']
2025-09-17 02:06:22,143 - ingestion_db - ERROR - load_raw_data:114 - Unexpected error processing file telecom_data.csv: cannot access local variable 'engine' where it is not associated with a value
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:131 - ==================================================
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:132 - INGESTION COMPLETE
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:133 - Total files processed: 1
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:134 - Successful: 0
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:135 - Failed: 1
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:136 - Total time taken: 0.00 minutes
2025-09-17 02:06:22,143 - ingestion_db - INFO - load_raw_data:137 - ==================================================
2025-09-17 20:52:19,898 - ingestion_db - INFO - create_connection:19 - Database connection established successfully
2025-09-17 20:55:51,982 - ingestion_db - INFO - create_connection:19 - Database connection established successfully
